test/jdk/javax/swing/regtesthelpers/JRobot.java

```diff
@@ -168,10 +190,13 @@ public void clickMouseOn(Component c, int buttons) {
 
     /**
      * Click the first mouse button in the center of the given Component
+     * <p>
+     * <b>Note:</b> This method uses EDT
```

Comment:
This instance still uses the old wording.

In [my comment](https://github.com/openjdk/jdk/pull/14354#discussion_r1242274807), I meant updating **all** the notes when I said about *being consistent*.

The `convertRectToScreen` method needs updating as well.